routers/api/v1/api.go

```diff
@@ -1495,6 +1495,8 @@ func Routes() *web.Route {
 					m.Post("/orgs", bind(api.CreateOrgOption{}), admin.CreateOrg)
 					m.Post("/repos", bind(api.CreateRepoOption{}), admin.CreateRepo)
 					m.Post("/rename", bind(api.RenameUserOption{}), admin.RenameUser)
+					m.Post("/badges", bind(api.UserBadgeOption{}), admin.AddUserBadges)
```

Comment:
No `get`?

Reply:
I kept that out of this PR as I was thinking of a more complex response that would return more details about the badge than just the slug. My focus for this PR was adding/removing badges (keeping things small ;))